### DISCUSSION TITLE
Deploying milvus gpu version issues

### DISCUSSION THREAD

**Discussion by Lunpopo** [milvus-io/milvus#27367] [Q&A and General discussion]
There was a problem when I deployed the Milvus GPU version.
see below：
```
[root@node2 home]# docker-compose up -d
validating /home/docker-compose.yaml: services.standalone Additional property resources is not allowed
```

The configuration file was downloaded from the official website.

and docker-compose version is v2.22.0.

what should i do?

**Comment by Lunpopo** [comment]
Additional information is as follows:
docker version: 24.0.2
docker-compose: v2.22.0

**Comment by yhmo** [comment]
“Additional property resources is not allowed”
We don't have a "resources" tag under "services.standalone" in the official docker-compose.yaml, did you manually add it?

**Reply by Lunpopo** [reply]
yeah, according to the official documentation, you need to add your own GPU configuration manually in docker compose yaml.
see: https://milvus.io/docs/install_standalone-gpu-docker.md
